CMakeLists.txt

```diff
@@ -1814,6 +1823,13 @@ if(HPX_WITH_DOCUMENTATION)
     PATTERN ".git" EXCLUDE)
 endif()
 
+
+if(HPX_WITH_VIM_YCM)
+  configure_file(${CMAKE_SOURCE_DIR}/tools/vim/.ycm_extra_conf.py
+               ${HPX_WITH_VIM_YCM}/.ycm_extra_conf.py @ONLY)
+  hpx_info("VIM YouCompleteMe: to enable automatic loading of configure file, add to your .vimrc option: \"let g:ycm_extra_conf_globlist = ['${HPX_WITH_VIM_YCM}/*']\"")
```

Comment:
It also seems that vim isn't the only compiler supported. See here:
https://atom.io/packages/you-complete-me